server/pageRenderer.js

```diff
@@ -110,7 +110,6 @@ export default function pageRenderer({
       </script>`
             : ''
         }
-        <script async src="https://js.stripe.com/v2/"></script>
```

Comment:
Isn't this still required by `react-stripe-checkout`? (aka. the package is still there)

Reply:
Hmm, I thought I removed it :open_mouth: